style.css

```diff
@@ -0,0 +1,90 @@
+.bg {
+  background: url(images/header-shapes-mobile.png) 80px 100px no-repeat;
+  height: auto;
+}
+
+.wrapper {
+  padding: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.mylogo {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.nav {
+  display: flex;
+  flex-direction: column;
+  width: 36px;
+  cursor: pointer;
+}
+
+.line {
+  border: 2px solid #6070ff;
+  width: 16px;
+  margin-bottom: 4px;
+}
+
+.myheader {
+  font-style: normal;
```

Comment:
- [ ] According to the Figma template, the font-style used is Poppins. Please use the Poppins font for all the characters in this project by importing the it from [here](https://fonts.google.com/specimen/Poppins) 👌